linkcheck/HtmlParser/htmlparse.y

```diff
@@ -988,35 +988,36 @@ MOD_INIT(htmlsax) {
         /* init error */
         PyErr_Print();
     }
-    if ((m = PyImport_ImportModule("linkcheck.HtmlParser")) == NULL) {
+    PyObject* h = NULL;
+    if ((h = PyImport_ImportModule("linkcheck.HtmlParser")) == NULL) {
         return MOD_ERROR_VAL;
     }
-    if ((resolve_entities = PyObject_GetAttrString(m, "resolve_entities")) == NULL) {
-        Py_DECREF(m);
+    if ((resolve_entities = PyObject_GetAttrString(h, "resolve_entities")) == NULL) {
+        Py_DECREF(h);
         return MOD_ERROR_VAL;
     }
-    if ((set_encoding = PyObject_GetAttrString(m, "set_encoding")) == NULL) {
+    if ((set_encoding = PyObject_GetAttrString(h, "set_encoding")) == NULL) {
         Py_DECREF(resolve_entities);
-        Py_DECREF(m);
+        Py_DECREF(h);
         return MOD_ERROR_VAL;
     }
-    if ((set_doctype = PyObject_GetAttrString(m, "set_doctype")) == NULL) {
+    if ((set_doctype = PyObject_GetAttrString(h, "set_doctype")) == NULL) {
         Py_DECREF(resolve_entities);
         Py_DECREF(set_encoding);
-        Py_DECREF(m);
+        Py_DECREF(h);
         return MOD_ERROR_VAL;
     }
-    Py_DECREF(m);
-    if ((u_meta = PyString_Decode("meta", 4, "ascii", "ignore")) == NULL) {
+    Py_DECREF(h);
+    if ((u_meta = PyUnicode_Decode("meta", 4, "ascii", "ignore")) == NULL) {
```

Comment:
This could also be PyUnicode_FromStringAndSize().

Reply:
Changed as well.